toolchain/sem_ir/name.cpp

```diff
@@ -39,6 +39,8 @@ static auto GetSpecialName(NameId name_id, bool for_ir) -> llvm::StringRef {
       return "_";
     case NameId::SpecialNameId::Vptr:
       return for_ir ? "vptr" : "<vptr>";
+    case NameId::SpecialNameId::CppOperator:
+      return "cpp_operator";
```

Comment:
```suggestion
      return for_ir ? "cpp_operator" : "<C++ operator>";
```
`cpp_operator` is fine as an IR name, but we shouldn't use it in (eg) diagnostics.

Reply:
Done, though not sure how to cover this with tests if you think that's necessary at this point.